File: Python/Program/Equipment.py
class Equipment:
    def __init__(self, name: str, purpose: str):
        self._name = name
        self._purpose = purpose

    def get_name(self):
        return self._name

    def set_name(self, name: str):
        self._name = name

    def get_purpose(self):
        return self._purpose

    def set_purpose(self, purpose: str):
        self._purpose = purpose

File: C++/Program/Equipment.cpp
#pragma once
#include <iostream>
#include <string>

using namespace std;

class Equipment {
private:
    string name;
    string purpose;

public:

    // Constructor
    Equipment(){}

    Equipment(string name, string purpose) {
        this->name = name;
        this->purpose = purpose;
    }

    // Getter and Setter for name
    string getName() {
        return name;
    }

    void setName(string name) {
        this->name = name;
    }

    // Getter and Setter for purpose
    string getPurpose() {
        return purpose;
    }

    void setPurpose(string purpose) {
        this->purpose = purpose;
    }
};